cmd/flags/cassandra/options_test.go

```diff
@@ -42,23 +43,27 @@ func TestOptions(t *testing.T) {
 }
 
 func TestOptionsWithFlags(t *testing.T) {
-	flags := flag.NewFlagSet("test", flag.ExitOnError)
-	opts := NewOptions()
-	opts.Bind(flags, "cas", "cas.aux")
-	flags.Parse([]string{
-		"-cas.keyspace=jaeger",
-		"-cas.servers=1.1.1.1,2.2.2.2",
-		"-cas.connections-per-host=42",
-		"-cas.max-retry-attempts=42",
-		"-cas.timeout=42s",
-		"-cas.port=4242",
-		"-cas.proto-version=3",
-		"-cas.socket-keep-alive=42s",
+	opts := NewOptions("cas", "cas.aux")
+
+	command := &cobra.Command{}
+	command.PersistentFlags().AddGoFlagSet(opts.CreateFlags())
+	viper.BindPFlags(command.PersistentFlags())
+	command.ParseFlags([]string{
```

Comment:
Would be nice to add a test with properties coming from the env var, specially because it documents what's the precedence order.